antenna/app.py

```diff
@@ -262,6 +273,27 @@ def extract_payload(self, req):
 
         return raw_crash, dumps
 
+    def get_throttle_result(self, raw_crash):
+        # FIXME(willkg): This means that legacy_processing and percentage
+        # are user-provided. We should sanitize them.
+
+        # If we don't have throttle results for this crash, then get them.
+        if 'legacy_processing' in raw_crash and 'percentage' in raw_crash:
+            return (
+                raw_crash['legacy_processing'],
+                'dontknow',
```

Comment:
We might want a different indicator here. Maybe "specified" or "preexisting" or something like that.